mne/viz/_brain/_timeviewer.py

```diff
@@ -254,6 +280,34 @@ def toggle_interface(self):
             else:
                 slider.Off()
 
+    def toggle_playback(self):
+        self.playback = not self.playback
+        self.time_elapsed = 0
+
+    def set_playback_speed(self, speed):
+        self.playback_speed = speed
+
+    def play(self):
+        if self.playback:
+            self.time_elapsed += 10
+            if self.time_elapsed >= self.playback_speed * 10:
+                times = self.brain._data['time']
+                time_idx = self.brain._data['time_idx']
+                time = times[time_idx] + 1. / self.playback_speed
+                idx = np.argmin(np.abs(times - time))
```

Comment:
After working out the user-time-to-data-time logic I let `interp1d` from scipy in `'linear'` mode do the work of picking the correct data.